VectorThrust.cs

```diff
@@ -62,7 +62,9 @@
 public const string LCDName = "%VectorLCD";
 
 // arguments, you can change these to change what text you run the programmable block with
-public const string standbyArg = "%standby";
+public const string standbytogArg = "%standby";
+public const string standbyonArg = "%standbyon";

```

Comment:
im changing it to %standbyenter and %standbyexit to avoid confusion with the end user.

since standby is like turning the script off, its confusing to have the word 'on' in the name